packages/application-extension/src/index.tsx

```diff
@@ -942,6 +942,23 @@ const propertyInspector: JupyterFrontEndPlugin<IPropertyInspectorProvider> = {
   }
 };
 
+const JupyterLogo: JupyterFrontEndPlugin<void> = {
+  id: '@jupyterlab/application-extension:logo',
```

Comment:
Looks like this plugin is missing the `autostart: true`.

Reply:
Fixed in https://github.com/jupyterlab/jupyterlab/pull/9274/commits/a851e913da6e977512f7eaf1f9e4a2317d24f2af